src/conmon.c

```diff
@@ -728,6 +701,91 @@ static gboolean conn_sock_cb(int fd, GIOCondition condition, gpointer user_data)
 	return G_SOURCE_REMOVE;
 }
 
+static gboolean conn_sock_cb(G_GNUC_UNUSED int fd, GIOCondition condition, gpointer user_data);
+
+static void sock_try_write_to_masterfd_stdin(struct conn_sock_s *sock)
+{
+	if (sock->remaining) {
+		ssize_t w;
+
+		w = write(masterfd_stdin, sock->buf + sock->off, sock->remaining);
```

Comment:
I would just compress this (I am looking to move away from declaring variables before defining them, c99 style)